### PR TITLE
user credit card update  component  was added

### PR DESCRIPTION
**Pull request by daisyzarzo**
relates to https://github.com/bespoken/dashboard/issues/964

### CODE REVIEW

**Comment by daisyzarzo** [comment]
image

File: src/components/BillCard/Payment.tsx [hunk 1]
```diff
@@ -86,11 +88,51 @@ class SplitFormC extends React.Component<any & SplitFormCProps, any> {
         }
     }
 
+    setStripeForModeJustUpdateCreditCard = async () => {
+        let creditCardObj: any;
+        const { stripeCustomerObjId } = this.props.userDetail;
+        let resultMessge: any = "";
```
Comment (jperata) [inline src/components/BillCard/Payment.tsx]: resultMessage

File: src/pages/MyAccountPage.tsx [hunk 2]
```diff
@@ -159,10 +162,19 @@ export class MyAccountPage extends React.Component<MyAccountPageProps, MyAccount
...
+    disablingDialog = async () => {
         await this.setState((prevState) => ({
             ...prevState,
             changePasswordDialogActive: false,
+            changeDeleteAccountDialogActive: false,
+            changeUpdateUserCreditCardDialogActive: false,
+        }));
+    }
+
+    handleUpdateUserCreditCardtDialogToggle = async () => {
```
Comment (jperata) [inline src/pages/MyAccountPage.tsx]: handleUpdateUserCreditCardDialogToggle

File: src/pages/MyAccountPage.tsx [hunk 3]
```diff
@@ -173,6 +185,40 @@ export class MyAccountPage extends React.Component<MyAccountPageProps, MyAccount
         }));
     }
 
+    handleUpdateUserCreditCardDialogToggle = async () => {
```
Comment (jperata) [inline src/pages/MyAccountPage.tsx]: duplicated function, it seems due to the typo in the previous function, the one that says "handleUpdateUserCreditCardtDialogToggle" should be removed

File: src/pages/MyAccountPage.tsx [hunk 4]
```diff
@@ -223,25 +271,36 @@ export class MyAccountPage extends React.Component<MyAccountPageProps, MyAccount
...
                         userId={userId} />
                 </Dialog>
+                <Dialog
+                    theme={DialogTheme}
+                    active={this.state.changeUpdateUserCreditCardDialogActive}
+                    onEscKeyDown={this.disablingDialog}
+                    onOverlayClick={this.disablingDialog} >
+                    <UpdateUserCreditCard
+                        handleUpdateUserCreditCard={this.handleUpdateUserCreditCard}
+                        handleCancelClick={this.handleUpdateUserCreditCardtDialogToggle}
```
Comment (jperata) [inline src/pages/MyAccountPage.tsx]: verify if you really need two functions and rename or change handleUpdateUserCreditCardDialogToggle

Comment (daisyzarzo) [reply src/pages/MyAccountPage.tsx]: this function handleUpdateUserCreditCard
shows the updates on the account page after card update

**Pull request merged**